modules/ossm-cr-threescale.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-cr-threescale_{context}"]
+
+= 3scale example
+
+Here is an example that illustrates the 3scale Istio Adapter parameters for the {ProductName} custom resource and a description of the available parameters with appropriate values.
+
+[source,yaml]
+----
+  threeScale:
+      enabled: false
+      PARAM_THREESCALE_LISTEN_ADDR: 3333
+      PARAM_THREESCALE_LOG_LEVEL: info
+      PARAM_THREESCALE_LOG_JSON: true
+      PARAM_THREESCALE_REPORT_METRICS: true
+      PARAM_THREESCALE_METRICS_PORT: 8080
+      PARAM_THREESCALE_CACHE_TTL_SECONDS: 300
+      PARAM_THREESCALE_CACHE_REFRESH_SECONDS: 180
+      PARAM_THREESCALE_CACHE_ENTRIES_MAX: 1000
+      PARAM_THREESCALE_CACHE_REFRESH_RETRIES: 1
+      PARAM_THREESCALE_ALLOW_INSECURE_CONN: false
+      PARAM_THREESCALE_CLIENT_TIMEOUT_SECONDS: 10
```

Comment:
@dfennessy I don't see the gRPC entries that were merged upstream here yet. Relates to https://github.com/Maistra/istio-operator/pull/92

Reply:
Thanks Philip,

Add the following:
...
PARAM_THREESCALE_LOG_JSON: true
`PARAM_THREESCALE_LOG_GRPC: false`
PARAM_THREESCALE_REPORT_METRICS: true
...
PARAM_THREESCALE_CLIENT_TIMEOUT_SECONDS: 10
`PARAM_THREESCALE_GRPC_CONN_MAX_SECONDS: 60`
...

Reply:
Updated.

Reply:
Still missing the description down below I believe?

Reply:
@PhilipGough There is no further description in the ossm-cr-threescale parameters module which is part of the Service Mesh instructions for creating the CR for Service Mesh installation. The 3scale Adapter documentation is now contained to its own folder in the following assembly/modules:

- service_mesh/threescale_adapter/threescale-adapter.adoc
- modules/ossm-threescale-integrate.adoc
- modules/ossm-threescale-cr.adoc
- modules/ossm-threescale-templates.adoc
- modules/ossm-threescale-manifests.adoc
- modules/ossm-threescale-routing.adoc
- modules/ossm-threescale-integration-settings.adoc
- modules/ossm-threescale-caching.adoc
- modules/ossm-threescale-authentication.adoc
- modules/ossm-threescale-metrics.adoc

Please review and advise on any changes to those files which are part of this PR.